proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,567 @@
+   do not serve the association's purpose or by disproportionately high
+   remuneration.
+
+## 4. Types of Membership and Members of the Association
+
+1. Membership is open to natural persons and legal bodies supporting the
+   association's aims. The following types of membership are possible:
+   1. **Core members**: Natural persons actively supporting the Association’s
+      purpose and aims. Primarily, they are expected to cooperate and
+      contribute, participate in the general assembly and exercise their voting
+      rights.
+   2. **Extraordinary members**: Natural persons and legal bodies supporting
+      the Association’s aims and are allowed to participate in the General
+      Assemblies, but have no voting rights and np obligations to cooperate.
+      This gives members the opportunity to pause their cooperation and 
+      contribution to Mixxx without necessarily losing their membership.
+   3. **Supporting Members**: Extraordinary members primarily supporting
+      through financial or material contributions. They have the opportunity to
+      participate in the General Assemblies without voting rights.
+1. Core membership is granted upon suggestion or application by the General
+   Assembly or through an internet vote of the core members. Criteria include
+   long-term commitment and contributions.
+2. A core member can change status to that of an extraordinary member by
+   submitting their request to the Board. Should an active member not fulfill
+   the above mentioned duties over a period of two consecutive general
+   meetings, the membership will automatically change into an extraordinary
+   membership. Upon request, the Board may grant an exception to this automatic
+   procedure.
+3. Membership terminates due to withdrawal, expulsion, death, or loss of legal
+   status. Withdrawal from the association shall take effect at the end of the
+   following calendar month in which the declaration of withdrawal was
+   submitted.
+4. The expulsion of a member may be decided by the General Assembly or by an
+   Internet Vote of core members in the event of a critical situation where an
+   amicable solution cannot be reached. The following reasons may cause an
+   expulsion:
+   * a) Serious violations of regulations in the Articles of Association,
+     the purpose or aim of the Association or the Code of Conduct.
+   * b) Non-payment of membership fees over a period of 12 months.
+   Member must be granted a two-week period before expulsion, in which it can 
+   justify their actions in writing. In the case of a), the Board may withdraw
+   access rights to the Association's infrastructure and the association and
+   prohibit the member from communicating on behalf of the association. If this
+   concerns a core member, this member retains their voting rights until
+   expulsion. 
+5. Members leaving the Association or if the Association is liquidated do not
+   have a right to reimbursement for contributed assets.
```

Comment:
This does not clearly regulate how the admission procedure works and in what form the declaration of admission should be submitted. Does a future member submit a written application form and then the board, the general meeting etc. decides?